scheme-service/src/main/java/uk/gov/justice/laa/fee/scheme/feecalculators/CalculateMediationFee.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.laa.fee.scheme.feecalculators;
```

Comment:
Suggestion inline with other package names:
```suggestion
package uk.gov.justice.laa.fee.scheme.feecalculator;
```